core/src/com/unciv/logic/map/MapParameters.kt

```diff
@@ -158,5 +158,6 @@ class MapParameters {
     }
 
     // For debugging and MapGenerator console output
-    override fun toString() = "($mapSize ${if (worldWrap)"wrapped " else ""}$shape $type, Seed $seed, $elevationExponent/$temperatureExtremeness/$resourceRichness/$vegetationRichness/$rareFeaturesRichness/$maxCoastExtension/$tilesPerBiomeArea/$waterThreshold)"
+    override fun toString() = if (name.isNotEmpty()) "\"$name\""
+        else "($mapSize ${if (worldWrap)"wrapped " else ""}$shape $type, Seed $seed, $elevationExponent/$temperatureExtremeness/$resourceRichness/$vegetationRichness/$rareFeaturesRichness/$maxCoastExtension/$tilesPerBiomeArea/$waterThreshold)"
```

Comment:
Please take the `if` out of here, I can barely read this

Reply:
I know it was there before but still

Reply:
:+1: - yea, a debug-only toString() also (unexpectedly) has the right to be readable